### DISCUSSION TITLE
How to predict single image using the save model

### DISCUSSION THREAD

**Discussion by CodesInTheShell** [keras-team/autokeras#1469] [Q&A] [answered]
I have trained cats and dogs for using ImageClassifier and save the model with model.save("model_autokeras.h5"), now when I use the model for prediction I only getting the RESULT below, I am expecting 2 classes (cats and dogs), I want to know the class predicted and the proba for each. Can someone point me to some documentation please.

```
from tensorflow.keras.models import load_model
import autokeras as ak
import numpy as np
from PIL import Imagepic="sample_dog.jpg"image_x = 180
image_y = 180img = Image.open(pic)
img = img.resize((image_x,image_y))
img = np.array(img) 
img = np.reshape(img, (-1, image_x, image_y, 3))model = load_model("model_autokeras.h5", custom_objects=ak.CUSTOM_OBJECTS)
predicted_y = model.predict(img)
print(predicted_y)
```

RESULT = [[0.9998584]]

**Comment by haifeng-jin** [comment, marked as answer]
For 2-class classification we are using a single output as the probability for the first class "1-p" would be the probability for the other class.
In your case, the probability would be 0.9998584 and (1-0.9998584) for cats and dogs.
